beacon_node/network/src/sync/block_lookups/mod.rs

```diff
@@ -341,12 +333,12 @@ impl<T: BeaconChainTypes> BlockLookups<T> {
         let request_state = R::request_state_mut(lookup).get_state_mut();
 
         match response {
-            Ok((response, seen_timestamp)) => {
+            Ok((response, peer_group, seen_timestamp)) => {
                 debug!(self.log,
                     "Received lookup download success";
                     "block_root" => ?block_root,
                     "id" => id,
-                    "peer_id" => %peer_id,
+                    "peer_group" => ?peer_group,
```

Comment:
Debug log for peer_group is quite verbose, we could probably implement `Display`

```
// %peer_id
1AnFkF14ZcbnBiT1UwXGbwhCqenCAhpr88EcdkYuYkFXkC

// ?peer_group
PeerGroup { peers: [PeerId("1AnFkF14ZcbnBiT1UwXGbwhCqenCAhpr88EcdkYuYkFXkC")] }

```